### ISSUE TITLE
How to connect to Data Node to fix long messages not logged

### ISSUE THREAD

**Issue by ili101** [Graylog2/docker-compose#61]
Hi, I want to try to apply one of these solutions, for example, https://github.com/Graylog2/graylog2-server/issues/873#issuecomment-446140324.
Using the "open-core" compose and default configurations on setup.
How do I connect to the Data Node to apply the fix? If I for example `curl -k -X GET "https://localhost:9200/_cluster/health?pretty=true"` I get "Authentication finally failed". Do I need to extract the certificate to connect or something similar? help appreciated.

**Comment by janheise** [comment]
@ili101 Hi, this is not so easy to accomplish. Is this a blocker for you? There are changes in the upcoming bugfix release for the DataNode and some more in 6.0 in that area that will help with making the type of changes you linked to.

**Comment by janheise** [comment]
Also, the 32k limit and how to "solve it" seems a bit controversial. Maybe you can solve this completely different?

**Comment by ili101** [comment]
Hi @janheise Unfortunately, as I can see all solutions to save more than 32k involve editing the settings with that API one way or another.
This is not a blocker for me as in the worst case I can truncate all fields at 32K in the code before sending it to Graylog. So if there is an abnormally long log it will unfortunately be partial but at last, the entire log will not be discarded.

**Comment by janheise** [comment]
@ili101 I hope it's not a misunderstanding: as far as I know, there is no solution to save more than 32k - it's only the question who cuts and ignores the rest. Personally, I think if you can cut it before putting it into the input, that is preferred because that means you are in control.

**Comment by ili101** [comment]
No, it saves the whole thing but doesn't let you search on its content. So you need to find the record by another field and then you get the whole content.

**Comment by SerheoReutty** [comment]
Hello. Can you give me a hint, please? 
I have deployed graylog from this repository. But I can't access the DataNode Api.
I have tried using ROOT_PASSWORD used to create SHASUM and PASSWORD_SECRET generated with `pwgen` both give an error 
Authentication finally failed
curl -k https://admin:<root_password>@localhost:9200/_cat/nodes? v
